Solutions/Marain.TenantManagement.Specs/Steps/ExceptionHandlingSteps.cs

```diff
@@ -0,0 +1,52 @@
+﻿// <copyright file="ExceptionHandlingSteps.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+namespace Marain.TenantManagement.Specs.Steps
+{
+    using System;
+    using Marain.TenantManagement.Exceptions;
+    using NUnit.Framework;
+    using TechTalk.SpecFlow;
+
+    [Binding]
+    public class ExceptionHandlingSteps
+    {
+        private readonly ScenarioContext scenarioContext;
+
+        public ExceptionHandlingSteps(ScenarioContext scenarioContext)
+        {
+            this.scenarioContext = scenarioContext;
+        }
+
+        [Then(@"an '(.*)' is thrown")]
```

Comment:
Do you have Roslynator installed? It normally complains about unnecessary use of `@`, which I think this is.